### PR TITLE
[TEST] - Test unit - Fix failing UI unit test main branch

### PR DESCRIPTION
**Pull request by utchoang**
### Description

Fix test unit & error lint

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [x] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by davidjumani** [comment]
@utchoang Can you fix the conflicts ? Thanks

**Comment by utchoang** [comment]
@blueorangutan ui

**Comment by utchoang** [comment]
@blueorangutan ui

**Comment by utchoang** [comment]
@blueorangutan ui

File: ui/tests/unit/views/AutogenView.spec.js [hunk 1]
```diff
@@ -2847,7 +2832,7 @@ describe('Views > AutogenView.vue', () => {
         }
 
         mockAxios.mockResolvedValue(mockData)
-        spyConsole.log = jest.spyOn(console, 'log').mockImplementation(() => {})
+        // spyConsole.log = jest.spyOn(console, 'log').mockImplementation(() => {})
```
Comment (DaanHoogland) [inline ui/tests/unit/views/AutogenView.spec.js:2835]: why keep this line in comment every where? we have version control in place.

Comment (utchoang) [reply ui/tests/unit/views/AutogenView.spec.js:2835]: Đó là một thiếu sót. Tôi đã quên xóa nó trong lần cam kết trước. Cảm ơn

Comment (utchoang) [reply ui/tests/unit/views/AutogenView.spec.js:2835]: That is a shortcoming. I forgot to remove it on the previous commit. Thanks

**Comment by utchoang** [comment]
@blueorangutan ui

**Comment by rhtyd** [comment]
UI based tests passed:
```
 DONE  No lint errors found!
> cloudstack-ui@1.0.0 test:unit /home/travis/build/apache/cloudstack/ui
> vue-cli-service test:unit
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db
[vue-jest]: Less are not currently compiled by vue-jest
 PASS  tests/unit/views/AutogenView.spec.js (22.462s)
 PASS  tests/unit/views/compute/MigrateWizard.spec.js
 PASS  tests/unit/components/widgets/Status.spec.js
 PASS  tests/unit/components/view/ActionButton.spec.js
=============================== Coverage summary ===============================
Statements   : 4.82% ( 814/16888 )
Branches     : 5.81% ( 509/8760 )
Functions    : 2.75% ( 151/5485 )
Lines        : 4.82% ( 808/16748 )
================================================================================
Test Suites: 4 passed, 4 total
Tests:       135 passed, 135 total
Snapshots:   0 total
Time:        83.779s
Ran all test suites.
```

**Review by nvazquez — APPROVED** [review]
LGTM

**Review by rhtyd — APPROVED** [review]

**Pull request merged**